tests/spread/plugins/go/classic/task.yaml

```diff
@@ -1,32 +1,24 @@
 summary: Build and run a basic Go snap
 
-# Classic snaps don't build on 14.04
```

Comment:
Is this comment no longer valid?

Reply:
Given we do not support _running_ on trusty with bases it is not even a test target.
We have to probably remove it as the default run and keep it for legacy triggers only (and a lxd/multipass smoke)